File: app/java/src/main/java/com/example/production/domain/model/inventory/IssueDetail.java
package com.example.production.domain.model.inventory;

import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;

/**
 * 払出明細データエンティティ
 */
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class IssueDetail {
    private Long id;
    private String issueNumber;
    private Integer lineNumber;
    private String itemCode;
    private BigDecimal issueQuantity;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: app/csharp/src/ProductionManagement.Domain/Models/Inventory/IssueDetail.cs
namespace ProductionManagement.Domain.Models.Inventory;

/// <summary>
/// 払出明細
/// </summary>
public class IssueDetail
{
    public long? Id { get; init; }

    public required string IssueNumber { get; init; }

    public required int LineNumber { get; init; }

    public required string ItemCode { get; init; }

    public required decimal IssueQuantity { get; init; }

    public DateTime CreatedAt { get; init; }

    public DateTime UpdatedAt { get; init; }
}
